shelley/design-spec/delegation_design_spec.tex

```diff
@@ -545,276 +573,205 @@ \section{Design of Delegation}
-and pointer addresses carry stake rights however.
+\begin{figure}[ht]
+  \includegraphics[width=\linewidth]{addresses.pdf}
+  \caption{Addresses and Credentials}
+  \label{fig:addresses}
+\end{figure}
 
-\subsubsection{Base Address}
-\label{base-address}
+\cref{fig:addresses} shows the anatomy of both payment and stake addresses. Let
+us first consider a \emph{Stake Address}: it contains a stake credential (which
+can be a key credential or script credential). A stake address is used for two
+purposes: delegating stake (see \cref{delegation-certificates}), and spending
+rewards accumulated in a \emph{reward account} (see \cref{reward-accounts}).
+
+\emph{Payment Addresses} have more structure: first of all, we have the two
+cases of Byron and Shelley addresses. The purpose of Byron addresses is
+backwards compatibility. Byron addresses have no notion of stake, so it is not
+possible to delegate from a Byron address; users will have to transfer funds to
+a Shelley address first.
+
+A Shelley address contains a payment credential (again, either a key or script
+credential). A transaction that consumes a UTxO entry with a Shelley address
+will need a witness for this payment credential in order to be validated. In
+addition, a Shelley address also contains a stake address reference. When
+calculating the stake distribution (see \cref{delegation-relations} for
+details), the system uses this reference to decide where to count the stake
+corresponding to the tokens in this address.
+
+There are three options for the stake address reference in a Shelley address: it
+can be provided \emph{by value}, i.e., just be the hash of a verification key or
+validator script. Shelley addresses that provide their stake address reference
+by value are sometimes also called \emph{base addresses}.
+
+There is also a more compact way of representing a stake address reference: since stake
+addresses need to be registered on the chain in order to be considered for the
+stake distribution (see \cref{stake-address-registration-certificates}), we can
+also include them \emph{by pointer}, pointing to the certificate that
+registered the stake address containing. Since the
+blockchain orders transactions, this pointer is quite small, containing only
+three numbers (slot index, transaction index within the block, and certificate
+index within the transaction). Shelley addresses that contain their stake
+ by pointer are also called \emph{pointer addresses}.
+
+The third possibility is to not have a stake address reference at all. In this case,
+the stake corresponding to the tokens in the address is not considered by the
+system at all (just as for a Byron address). In particular, it will not be
+counted towards the active stake (\cref{overall-stake-distribution}), and so
+will not slow down chain growth. Users who do not wish to contribute to the PoS
+protocol should use this option. Such addresses are also called \emph{enterprise
+  addresses}.
```

Comment:
Maybe move away from enterprise addresses now?

Reply:
Given that we have used the term for a long time, and it has permeated, I think the document should at least explain the term at this point. But we can try to avoid using it too much across the document and elsewhere.